### PR TITLE
[ResourceBundle][TranslationBundle] Fix sorting on translated properties

### PR DESCRIPTION
**Pull request by cdaguerre**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | none
| License       | MIT
| Doc PR        | none

Check if properties are translated when sorting (as it's already done when filtering).

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@cdaguerre Nice, could you have a look at the failing build?

File: src/Sylius/Bundle/TranslationBundle/Doctrine/ODM/MongoDB/TranslatableResourceRepository.php [hunk 1]
```diff
@@ -80,19 +78,43 @@ protected function applyCriteria(QueryBuilder $queryBuilder, array $criteria = n
...
+                    ->field($this->getPropertyName($property))->equals($value)
                 ;
             }
         }
     }
+
+    /**
+     * {inheritdoc}.
+     */
+    protected function applySorting(QueryBuilder $queryBuilder, array $sorting = null)
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/TranslationBundle/Doctrine/ODM/MongoDB/TranslatableResourceRepository.php]: Why not simply initialize ```$sorting``` as an empty array?

```php
protected function applySorting(QueryBuilder $queryBuilder, array $sorting = array())
```

We could now simplify method by deleting check for null variable.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/TranslationBundle/Doctrine/ODM/MongoDB/TranslatableResourceRepository.php]: This is convention from Doctrine, they do the same.

**Comment by pjedrzejewski** [comment]
Thank you Christian! Nice work! :+1:

**Pull request merged**